language/Portuguese_Brazil/strings.po

```diff
@@ -2023,17 +2026,17 @@ msgstr "Chefes"
 #: src/frontend/mame/ui/dirmenu.cpp:61
 msgctxt "path-option"
 msgid "Artwork Previews"
-msgstr "Pré-Visualização dos Trabalhos das Artes"
+msgstr "Pré-Visualização do Trabalho de Arte"
 
 #: src/frontend/mame/ui/dirmenu.cpp:62
 msgctxt "path-option"
 msgid "Select"
-msgstr ""
+msgstr "Select"
```

Comment:
This message is the display name for the setting for the folder that holds screenshots of game mode selection or character selection screens – is this the most appropriate translation?  You translated the display title for these below images as “Selecionar”.

Reply:
Fixed.